### ISSUE TITLE
local git exercise broken on windows

### ISSUE THREAD

**Issue by SepplL** [pep-dortmund/toolbox-workshop#431]
Windows adds local zone identifiers to `repo` therefore blocking any git communication due to local zone identifiers errors.

This only happens, when files were touched by the explorer - if all is kept within the WSL the issue should not occur.

**Comment by aknierim** [comment]
We could create another template repository that contains the exercise. That way we can also teach using GitHub

**Comment by SepplL** [comment]
> We could create another template repository that contains the exercise. That way we can also teach using GitHub

Do we want to do that? I could work the template out and change the exercise accordingly for the next workshop in the coming days.

**Comment by SepplL** [comment]
Yes exactly. Additionally they can practice the template usage here as we will use it for the entire "Praktikum-template" later.

Also I think there is no harm in keeping the offline version, in case someone wants to check how these files could look like.
